### PR TITLE
SP-33 updated php 7.1 to be latest, added xdebug and custom .ini settings

### PR DESCRIPTION
**Pull request by mcolledge91**

### CODE REVIEW

**Comment by mcolledge91** [comment]
I have already manually checked that all build commands in the `buildspec.yml` should be expected to work fine

File: 7.1/dev/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+RUN apk update \
+    && apk add --no-cache $PHPIZE_DEPS bzip2-dev freetype-dev libjpeg-turbo-dev libpng-dev \
+    && docker-php-source extract \
+    && docker-php-ext-configure gd \
+        --with-gd \
+        --with-freetype-dir=/usr/include/ \
+        --with-png-dir=/usr/include/ \
+        --with-jpeg-dir=/usr/include/ \
+    && docker-php-ext-install bcmath bz2 gd pdo_mysql zip \
+	&& pecl install xdebug-2.5.0 \
```
Comment (Tyrannogina) [inline 7.1/dev/Dockerfile]: When I tried this locally, I don't remember why but I changed it to 2.6.0. Is there a reason for using this particular version?
Also, isn't indentation too deep here?

Comment (mcolledge91) [reply 7.1/dev/Dockerfile]: @Tyrannogina requires one extra setting for your IDE, but all seems to be working now with the latest version of xdebug

File: 7.2/Dockerfile [hunk 2]
```diff
@@ -17,4 +21,4 @@ RUN apk update \
     && apk del $PHPIZE_DEPS \
     && rm -rf /tmp/* /var/cache/apk/*
 
-MAINTAINER Pod-Point <software@pod-point.com>
+#TODO out of date, use 7.1 for now
```
Comment (LeeBrooks3) [inline 7.2/Dockerfile]: we don't tend to commit TODO's

File: README.md [hunk 3]
```diff
@@ -1,30 +1,60 @@
...
+This should allow you to use the image without `ssm-fetch`
+
+## WIP
+
+Just a word of warning, this codebase is still in development, here are a few things that still need doing
+
+* prod ready images
+* distribution of scripts to images without duplication
+* more php versions
+* feature documentation
```
Comment (LeeBrooks3) [inline README.md]: again I wouldn't add this into the README for now, just make a note somewhere else (maybe raise an issue on the repo?)

Comment (mcolledge91) [reply README.md]: Sure thing, was torn between including it or not. Wouldn't want anyone accidentally using unready images - however a github issue sounds like a great idea

Comment (mcolledge91) [reply README.md]: @LeeBrooks3 updated and issues raised

**Pull request merged**